t/03-auth.t

```diff
@@ -13,12 +13,12 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
+use Test::Most;
 use Mojo::Base -strict;
```

Comment:
Both activate strictures here.

Reply:
I know, should we then just write:

```
use Test::Most;
use Mojo::Base;
```

?

Reply:
No, i guess in this case you'd just remove `Mojo::Base`. I think in all the tests it's only imported to activate strictures. You just need to watch out for the other things `use Mojo::Base -strict` does. Maybe one or two of the tests actually depend on the `utf8` pragma for example.
```perl
# "use Mojo::Base -strict;" activates these things
use strict;
use warnings;
use utf8;
use feature ':5.10';
use mro;
use IO::Handle ();
```

Reply:
ah, good. We'll see :)

Reply:
tweaked imports in some modules to provide the things that were now missing. Seems to be all good so far. Need to see results from all tests from CI, especially full stack now. Thx

Reply:
awesome, t/ui/15-comments.t failed with unicode errors :) https://app.circleci.com/pipelines/github/os-autoinst/openQA/2496/workflows/c4768f34-d2ce-4745-ae34-f06e6229b04c/jobs/23466/parallel-runs/0/steps/0-108

Reply:
`use utf8;`

Reply:
Yes, I know. I am going over all failed test cases and adding the relevant parts. Thx